osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/application/mapping/LowVoltageMeasurementToAnalogList.java

```diff
@@ -30,7 +30,7 @@
       final int length) {
     return (Class<T>)
         Stream.of(LowVoltageMeasurementType.class, LowVoltageMetaMeasurementType.class)
-            .filter(clazz -> clazz.getEnumConstants().length == length)
+            .filter(clazz -> clazz.getEnumConstants().length == length || 41 == length)
```

Comment:
Isn't it better to check for either 41 or 52 elements? Now it is possible to send a new version without being checked.

Reply:
Implemented a different solution, no longer requiring a check on the hard-coded number.